docs/docs/guides/integrate/login-ui/typescript-repo.mdx

```diff
@@ -0,0 +1,62 @@
+---
+title: Typescript Repository
+sidebar_label: Typescript Repository
+---
+
+To replace our current golang built login UI and showcase the use of our new resource and OIDC APIs, we've created the [Typescript Repository](https://github.com/zitadel/typescript).
+The Typescript Repository contains all TypeScript and JavaScript packages and applications you need to create your own ZITADEL Login UI.
+The repo implements the [OIDC standard](./oidc-standard)
+
+## Included Apps And Packages
+
+- `login`: The login UI used by ZITADEL Cloud, powered by Next.js
```

Comment:
Its not yet used by the ZITADEL Cloud, should we write something like that should be the login for ZITADEL cloud in the future? (As soon as we have the functionality ready)

```suggestion
- `login`: The login UI used by ZITADEL Cloud, powered by Next.js
```

Reply:
I've stated that it is the future login UI now 👍